Ship queryTemplate to v1.0 doc (#20829)
microsoftgraph/microsoft-graph-docs: api-reference/v1.0/api/search-query.md

## Examples

### Example 1: Basic call to perform a search request

> The following example shows how to perform a search to get expected connector items.

Reviewer: ```suggestion
The following example shows how to search for expected connector items.
```

#### Request

```HTTP

> POST https://graph.microsoft.com/beta/search/query

Reviewer: Quick question, why did you change the version?  Thanks.

Author: Thank you, that's a mistake. Corrected.
